public/app/features/live/data/StreamingDataFrame.test.ts

```diff
@@ -1,8 +1,71 @@
 import { reduceField, ReducerID, getFieldDisplayName, DataFrame, FieldType, DataFrameJSON } from '@grafana/data';
 import { StreamingFrameAction, StreamingFrameOptions } from '@grafana/runtime';
-import { getStreamingFrameOptions, StreamingDataFrame } from './StreamingDataFrame';
+import { closestIdx, getStreamingFrameOptions, StreamingDataFrame } from './StreamingDataFrame';
 
 describe('Streaming JSON', () => {
+  describe('closestIdx', function () {
```

Comment:
should the changes here get their own PR (and committed soon?)

Reply:
https://github.com/grafana/grafana/pull/45162